src/app/content/utils/urlUtils.ts

```diff
@@ -98,5 +98,18 @@ export const toRelativeUrl = (from: string, to: string) => {
   const commonParts = getCommonParts(parsedFrom, parsedTo.slice(0, -1));
 
   return '../'.repeat(Math.max(0, parsedFrom.length - commonParts.length - 1))
-    + parsedTo.slice(commonParts.length).join('/');
+  + parsedTo.slice(commonParts.length).join('/');
+};
+
+export const fromRelativeUrl = (from: string, to: string) => {
```

Comment:
I think, this should check where `to` and `from` intersect, and return `to` + (`from` - everything before intersection in `from`)

Reply:
they won't literally string intersect because `to` is relative. you'll have to break the `from` into parts and remove pieces from the end for every `../` in the `to`, then add the remainder of the `to` to whatever is left. very similar logic to the `toRelativeUrl` function but backwards